pom.xml

```diff
@@ -105,6 +105,7 @@
         <module>services</module>
         <module>archetypes</module>
         <module>jbang</module>
+        <module>test-base</module>
```

Comment:
I'm thinking that maybe this one should be build first, so we can use this test module on the code.